common/dateHelpers.ts

```diff
@@ -0,0 +1,47 @@
+import moment from 'moment'
```

Comment:
I think moment is deprecated now. 
 I know other projects are using  https://date-fns.org/

Reply:
Turns out this file wasn't needed!